consensus/consensus_service.go

```diff
@@ -510,16 +511,16 @@ func (consensus *Consensus) UpdateConsensusInformation() Mode {
 
 	for _, key := range pubKeys {
```

Comment:
If only one key is in committee, we don't need to use other keys to sign and send transactions. Can use a data structure to hold the keys in committee only for the current epoch.